### PR TITLE
[8.0][WIP] l10n_it_account

### PR DESCRIPTION
**Pull request by dcorio**
l10n_it_account has been created this morning during the sprint because the vat_statement_type is needed by other localization modules which should not depend on account_vat_period_end_statement.
so we moved it to a new module (l10n_it_account).

we're going to use this new module to host other accounting helpers (fields, configurations, defaults, ...)

### CODE REVIEW

File: l10n_it_account/model/account.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import fields, models
```
Comment (tafaRU) [inline l10n_it_account/model/account.py]: Replace `openerp.osv` by `openerp`

**Comment by eLBati** [comment]
LGTM :+1:

**Comment by tafaRU** [comment]
@dcorio is still WIP?

**Comment by dcorio** [comment]
@tafaRU it is but @eLBati was suggesting to merge it to l10n-italy if it's fine right now and we got the 3 :+1:  of course so we can improve it by creating PR against l10n-italy

**Comment by dcorio** [comment]
@tafaRU still WIP

File: l10n_it_account/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2010 OpenERP Italian Community
```
Comment (tafaRU) [inline l10n_it_account/__init__.py:4]: Please align copyrights to the other headers.

File: l10n_it_account/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    'description': "Accounting-related modules helpers""",
+    'author': "Agile Business Group,Abstract,Odoo Community Association (OCA)",
+    'website': 'http://www.odoo-italia.org',
+    'license': 'AGPL-3',
+    "depends": ['base', 'account'],
+    "data": [
+        'views/account_view.xml',
+        'reports/account_reports_view.xml',
+    ],
+    "qweb": [],
```
Comment (tafaRU) [inline l10n_it_account/__openerp__.py]: Please remove empty keys.

File: l10n_it_account/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    "demo": [],
+    "test": [],
+    "active": False,
```
Comment (tafaRU) [inline l10n_it_account/__openerp__.py]: This key is deprecated and it's an alias for `auto_install` whose default value is already `False`. So please remove it.

**Comment by tafaRU** [comment]
> @tafaRU still WIP

Ok, but I started to spot some issues concerning OCA guidelines.

**Comment by dcorio** [comment]
@tafaRU and i appreciate that :) :+1:

**Pull request closed without merging**